modules/extensions-spring-support/src/main/java/org/mule/module/extension/internal/capability/xml/schema/SchemaDocumenter.java

```diff
@@ -70,19 +70,19 @@ void document(Declaration declaration, TypeElement extensionElement, RoundEnviro
     {
         declaration.setDescription(getJavaDocSummary(processingEnv, extensionElement));
         documentConfigurations(declaration, extensionElement, roundEnvironment);
-        documentOperations(declaration, roundEnvironment);
+        documentOperations(roundEnvironment, processingEnv, declaration);
     }
 
 
-    private void documentOperations(Declaration declaration, RoundEnvironment roundEnvironment)
+    private void documentOperations(RoundEnvironment roundEnv, ProcessingEnvironment processingEnv, Declaration declaration)
     {
-        final Map<String, ExecutableElement> methods = getOperationMethods(roundEnvironment);
+        final Map<String,Element> methods = getOperationMethods(roundEnv, processingEnv);
 
         try
         {
             for (OperationDeclaration operation : declaration.getOperations())
             {
-                ExecutableElement method = methods.get(operation.getName());
+                Element method = methods.get(operation.getName());
```

Comment:
any reason why forcing the `continue` instead of comparing with `!= null` and putting the logic in there?